make/Makefile.dev

```diff
@@ -0,0 +1,89 @@
+DOCKER_REPO?=quay.io/openshiftio
```

Comment:
can we change this docker org?

Reply:
Yes, let's do this when we know what it is..